README.md

```diff
@@ -138,8 +138,36 @@ end.
    make test
    ```
 
+### 2.3.1 GitHub credentials
 
-### 2.3.1 AWS credentials
+Integration tests require a GitHub personal access token to be configured.
+
+1. Log into your account on https://github.com/. Click your user icon and 
+navigate to *Settings* -> *Developer settings* -> *Personal access tokens*
+
+2. Click *Generate new token*
+
+3. Enter an approproate description such as "Integration tests for Azul"
+
+4. Select *No expiration*
+
+5. Do not select any scopes
+
+6. Click *Generate token* and copy the resulting token
+
+7. Edit the `deployments/.active/environment.local.py` file and modify the
+   `GITHUB_TOKEN` variable: 
+
+   ```
+   'GITHUB_TOKEN': '<the token you just copied>'
+   ```
+   
+   Do not add the token to any `environment.py` files.
+
+8. Repeat the previous step for any deployments to intend to use for running the
```

Comment:
```suggestion
8. Repeat the previous step for any deployments you intend to use for running the
```

Reply:
Looks like @jessebrennan missed this. I guess I should have just requested this change through the normal procedure.

Reply:
Sorry, I did see this at one point, but forgot when it actually came to squashing the PR.